recipes/usalign/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "US-align" %}
+{% set version = "2024.07.30" %}
+{% set org = "pylelab" %}
```

Comment:
This variable is used just once. IMO it would be clearer if it is inlined.

Reply:
OK